.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+  - "2.6"
+  - "2.7"
+  - "3.3"
+  - "3.4"
+sudo: false
+install: true
+script: true
```

Comment:
The red circle there is indicating that this file is missing a new line character at the end of the last line in the file.